config/appstructure.go

```diff
@@ -138,6 +138,9 @@ func (structure *GeneratedAppStructure) printSummary(summary SummaryType, fileNa
 func evaluateActionList(actions []scenario.Action, includeRaw bool, summary SummaryType) []scenario.Action {
 	appStructureScenario := make([]scenario.Action, 0, len(actions))
 	for _, act := range actions {
+		if act.Disabled {
```

Comment:
Good catch!

Reply:
yes somehow forgot about that the first time around